specification/src/main/asciidoc/webprofile/Introduction.adoc

```diff
@@ -93,7 +93,8 @@ from each grouping.
 For the first one, the Jakarta EE Platform
 specification mandates support for the “ _java:_ ” naming context in all
 profiles. Consequently, Web Profile products must support it. For a
-similar reason, all Web Profile 8 products must support the Java(TM) Platform, Standard Edition 8 API.
+similar reason, all Web Profile 9 products must support the Java(TM) Platform, Standard Edition 8 API,
```

Comment:
I don't think we said that products have to support JDK 8. We said apis have to be compiled for JDK 8.

Reply:
Agree, @smillidge.  Maybe this can be simplified to saying "..all Web Profile 9 APIs must support the Java(TM) Platform, Standard Edition 8 API, and all Web Profile 9 products must run on Java(TM) Platform, Standard Edition 11 runtime.